mxcube3/routes/samplecentring.py

```diff
@@ -368,10 +368,19 @@ def centre_3_click():
         :statuscode: 200: no error
         :statuscode: 409: error
     """
-    data = sviewutils.start_manual_centring()
 
-    resp = jsonify(data)
-    resp.status_code = 200
+    try:
+        data = sviewutils.start_manual_centring()
+    except Exception as ex:
+        resp = (
```

Comment:
I suggest 'early return' on error. I.e. the whole code will be:

    try:
        data = sviewutils.start_manual_centring()
    except Exception as ex:
        return (
            "Could not move motor %s" % str(ex),
            406,
            {"Content-Type": "application/json", "msg": str(ex)},
        )

    resp = jsonify(data)
    resp.status_code = 200

    return resp

IMHO, this style of returning on errors is easier to read.

Reply:
We can continue this discussion on our next meeting, but I don't really agree that the "early return" makes the code easier to read. I'm more a fan of only one return per function, except for in certain cases when it's clearly more readable with "early returns" I don't think this is such a case. I'm in anyways happy to discuss it and I happy to use whatever style we agree on :) 

What do you think about that ?